program-runtime/src/timings.rs

```diff
@@ -49,15 +65,10 @@ impl ExecuteDetailsTimings {
             .saturating_add(other.data_size_changed);
         for (id, other) in &other.per_program_timings {
             let program_timing = self.per_program_timings.entry(*id).or_default();
-            program_timing.accumulated_us = program_timing
-                .accumulated_us
-                .saturating_add(other.accumulated_us);
-            program_timing.accumulated_units = program_timing
-                .accumulated_units
-                .saturating_add(other.accumulated_units);
-            program_timing.count = program_timing.count.saturating_add(other.count);
+            program_timing.accumulate_program_timings(other);
```

Comment:
here was the bug, didn't account for the newly added `errored_txs_compute_consumed`

Reply:
good catch!